test/queue_audit_test.rb

```diff
@@ -57,7 +57,7 @@ class QueueAuditTest < Minitest::Test
           Timecop.freeze(Time.now) do
             @audit.complete!
 
-            assert_equal Time.now, @audit.completed_at
+            assert_equal Time.now.to_i, @audit.completed_at.to_i
```

Comment:
Why do we set the precision to 6 digits in the schema and then use 0 digits when we compare timestamps?

Reply:
These were separate changes: I set precision to 6 for consistency across tests, and then decided that `to_i`’s precision is good enough for these tests. Do you think we should test these more precisely?

Reply:
The `to_i` conversion should not be necessary, especially inside a `Timecop.freeze` block. If the `to_i` is necessary for making _some_ of the tests pass, I suspect those tests fail once in a while when rounding `3.999997` to 3 and `4.000002` to 4.

Reply:
Reworked it to use `Time#floor`. Maybe there’s a better way to do this? The problem is that while in the database we now only have precision 6, `Time.now` still has additional nanoseconds, so we have to either cut them off or make a comparison less strict.